giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/service/impl/FileContentHelper.java

```diff
@@ -33,34 +45,32 @@
         } catch (IOException e) {
             logger.error("Could not read file.", e);
             return null;
+        } catch (RestClientResponseException e) {
+            logger.error("REST Exception", e);
+            return null;
         }
     }
     
     /* (non-Javadoc)
      * @see edu.asu.giles.service.impl.IFileSystemHelper#getFileContentFromUrl(java.net.URL)
      */
     @Override
-    public byte[] getFileContentFromUrl(URL url) throws IOException {
-        URLConnection con = url.openConnection();
+    public byte[] getFileContentFromUrl(URL url) throws IOException, RestClientResponseException {
```

Comment:
Does this always work? This method is also called to get the content of a file (from the method above) and it makes no sense at all why getting a file from the file system would throw a restclientexception.

Also, even if it works, this class is not written to be exclusively used with Nepomuk. Although it only connects to the file system or Nepomuk, it could be used to connect to somewhere else in the future. In other words, it has to be choosable, which token (if any at all) should be used.